.github/workflows/eas.yml

```diff
@@ -150,6 +150,114 @@ jobs:
             echo "A simulator was already found for ${{ steps.continuos_deploy.outputs.ios-fingerprint }}."
           fi
 
+      - name: Set QR Code URLs
+        id: set-urls
+        run: |
+          # Assume update-output contains the JSON output as a string
+          UPDATE_OUTPUT='${{ steps.continuos_deploy.outputs.update-output }}'
+
+          # Parse the JSON output to extract iOS and Android update IDs
+          IOS_UPDATE_ID=$(echo "$UPDATE_OUTPUT" | jq -r '.[] | select(.platform == "ios") | .id')
+          ANDROID_UPDATE_ID=$(echo "$UPDATE_OUTPUT" | jq -r '.[] | select(.platform == "android") | .id')
+
+          # Construct the Update QR code URLs using the update IDs
+          IOS_UPDATE_QR_URL="https://qr.expo.dev/eas-update?updateId=${IOS_UPDATE_ID}&appScheme=betterangels"
+          ANDROID_UPDATE_QR_URL="https://qr.expo.dev/eas-update?updateId=${ANDROID_UPDATE_ID}&appScheme=betterangels"
+
+          # Construct the Build QR code URLs using iOS and Android build IDs
+          IOS_BUILD_QR_URL="https://expo.dev/accounts/better-angels/projects/betterangels/builds/${{ steps.continuos_deploy.outputs.ios-build-id }}"
+          ANDROID_BUILD_QR_URL="https://expo.dev/accounts/better-angels/projects/betterangels/builds/${{ steps.continuos_deploy.outputs.android-build-id }}"
+
+          # Output all QR code URLs to the environment
+          {
+            echo "IOS_UPDATE_QR_URL=${IOS_UPDATE_QR_URL}"
+            echo "ANDROID_UPDATE_QR_URL=${ANDROID_UPDATE_QR_URL}"
+            echo "IOS_BUILD_QR_URL=${IOS_BUILD_QR_URL}"
+            echo "ANDROID_BUILD_QR_URL=${ANDROID_BUILD_QR_URL}"
+          } >> $GITHUB_ENV
+
+          # Log URLs for visibility in action logs
+          echo "iOS Update QR Code URL: ${IOS_UPDATE_QR_URL}"
+          echo "Android Update QR Code URL: ${ANDROID_UPDATE_QR_URL}"
+          echo "iOS Build QR Code URL: ${IOS_BUILD_QR_URL}"
+          echo "Android Build QR Code URL: ${ANDROID_BUILD_QR_URL}"
+
+      - name: Post to Slack
+        uses: slackapi/slack-github-action@v1.24.0
+        # if: github.ref == 'refs/heads/main'
```

Comment:
did you mean to leave the commented out or you wanna check if it's `main`

Reply:
Yes PR is still in draft but i need to move it out of draft for ci to run fully

Reply:
aah ok. assumed it was ready